src/platform/forms-system/src/js/fields/ArrayField.jsx

```diff
@@ -451,10 +453,8 @@ export default class ArrayField extends React.Component {
                     <VaModal
                       clickToClose
                       status="warning"
-                      modalTitle={`Are you sure you want to remove this ${uiOptions.itemName ||
-                        'item'}?`}
-                      primaryButtonText={`Yes, remove this ${uiOptions.itemName ||
-                        'item'}`}
+                      modalTitle={`Are you sure you want to remove this ${itemName}?`}
+                      primaryButtonText={`Yes, remove this ${itemName}`}
```

Comment:
No longer need the `|| 'item'` logic because we are reusing the `itemName` variable that already has that in its instantiation